tests/util/DummyClam.php

```diff
@@ -50,26 +50,36 @@ protected function handleConnection($connection){
 		$isAborted = false;
 		$command  =  fread($connection, 10);
 		//starts from INSTREAM\n from the first packet;
-
+		echo 'handling connection'. PHP_EOL;
 		if ($command !== "nINSTREAM\n"){
+			echo 'not instream'. PHP_EOL;
 			return;
 		}
 		//echo $command;
 		do {
+			echo 'looping'. PHP_EOL;
 			$binaryChunkSize = @fread($connection, 4);
 			$chunkSizeArray = unpack('Nlength', $binaryChunkSize);
 			$chunkSize = $chunkSizeArray['length'];
+			echo 'chunksize = ' .$chunkSize. PHP_EOL;
 			if ($chunkSize === 0){
 				break;
 			}
 
 			$dataChunk = '';
+			$limit = 10;
 			do {
+				echo 'reading dataChunk' . PHP_EOL;
 				$dataChunk .= @fread($connection, $chunkSize);
-			} while (is_resource($connection) && strlen($dataChunk) !== $chunkSize);
+				echo 'dataChunk size '.strlen($dataChunk). PHP_EOL;
+				$limit--;
+			} while (is_resource($connection) && strlen($dataChunk) !== $chunkSize && $limit > 0);
+			if ($limit === 0) break;
```

Comment:
@VicDeo 
this very dirt hack of limiting will make the PR green - but it was more intended to abort termination to not fill drone logs

Reply:
well but it breaks for a different test then:

```
chunksize = 8192
reading dataChunk
dataChunk size 8188
reading dataChunk
dataChunk size 16380
reading dataChunk
dataChunk size 16422
reading dataChunk
dataChunk size 16422
reading dataChunk
dataChunk size 16422
reading dataChunk
dataChunk size 16422
reading dataChunk
dataChunk size 16422
reading dataChunk
dataChunk size 16422
reading dataChunk
dataChunk size 16422
reading dataChunk
dataChunk size 16422
16422 524288
warning - limit reached during fread - using abortScanned OK
```